routes/user.js

```diff
@@ -4,7 +4,7 @@ const mongoose = require("mongoose");
 const User = require("../models/user.model");
 const { requireUserAuth } = require("../utils");
 const { multerUploads, uploadImagesToAWS } = require("./photos");
-const {sendVerificationEmail} = require('../helpers/account.helper')
+const { sendVerificationEmail } = require("../helpers/account.helper");
```

Comment:
the emails are now all in `../helpers/emails.helper` btw

Reply:
Fixed!